sysid-application/src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -38,8 +38,8 @@ Analyzer::Analyzer(glass::Storage& storage, wpi::Logger& logger)
   m_presets["WPILib (Pre-2020)"] = presets::kWPILibOld;
   m_presets["CANCoder"] = presets::kCTRECANCoder;
   m_presets["CTRE"] = presets::kCTREDefault;
-  m_presets["REV (Brushless)"] = presets::kREVBrushless;
-  m_presets["REV (Brushed)"] = presets::kREVBrushed;
+  m_presets["REV (NEO Built-in Encoder)"] = presets::kREVNEOBuiltIn;
+  m_presets["REV (Non NEO Encoder)"] = presets::kREVNonNEO;
```

Comment:
I think REV's docs call it "alternate encoder mode", so the string would be "REV (Alternate Encoder)".

Reply:
But you could still use an Encoder on the Encoder port that technically isn't an Alternate encoder (iirc Alternate is for data port encoders) so I thought NEO vs Non NEO would make the difference more clear.

Reply:
The encoder port is only intended for the communitation encoder, isn't it? The firmware wouldn't know whether a different motor was plugged into the encoder port, so it would apply the same filtering to whatever's plugged in.

Reply:
The thing is that we support a Quadrature Encoder (for brushed mode) plugged into the Encoder port, so that's where that naming could get confusing. Though I do agree that doing NEO vs Non NEO is a point of confusion for the 550.

Reply:
What about "REV (Brushless Built-in Encoder)", "REV (Brushed Encoder Port)", and "REV (Data Port)"?

Reply:
Yeah that works